### DISCUSSION TITLE
我这个怎么没有标签呀

### DISCUSSION THREAD

**Discussion by Uyoahz26** [walinejs/waline#1274] [Q&A] [answered]
从valine迁移过来的，页面不显示专属标签
![image](https://user-images.githubusercontent.com/42881988/183878248-c8cc61db-71c8-4d81-bddd-2dbf4c534a8d.png)
![image](https://user-images.githubusercontent.com/42881988/183878273-d23710c9-143c-499a-b080-070635dd0d27.png)

**Comment by Uyoahz26** [comment, marked as answer]
对不起 我是傻叉，没登录，登录了就有了

**Reply by w0oo** [reply]
🤣
